CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -1,3 +1,14 @@
+# v1.10.2 - Changelog since v1.10.1
+
+### Bug or Regression
+
+- Add libraries needed for determining XFS volume expansion ([#1204](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1204), [@nberlee](https://github.com/nberlee))
+
+### Cleanup
+
+- Add metrics for CSI server side error count ([#1237](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1237), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
```

Comment:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1232 
This one is also cherry-picked to 1.10?

Reply:
Hmm, it didn't appear in the release notes generation. Let me look.

Reply:
Ah, it got dropped from the cherry-pick. Thanks for catching. Updated.